osbrain/tests/test_agent.py

```diff
@@ -50,7 +50,9 @@ def test_early_agent_proxy(nsaddr):
     # Locate agent now
     a0 = Proxy('a0', timeout=3.)
     # Just check agent is ready
-    assert a0.ping() == 'pong'
+    safe_status = a0._safe
+    assert a0.unsafe.ping() == 'pong'
+    a0._safe = safe_status
```

Comment:
As a side note, completely unrelated, you can use:

    ```python
    print('Hello world')
    ```

If you want the code in your comments to get highlighted as Python code.